articles/man-in-the-middle-attack/index.md

```diff
@@ -0,0 +1,68 @@
+- [SSL Stripping](http://techgenix.com/Understanding-Man-in-the-Middle-Attacks-ARP-Part4/) - It minimizes an HTTPS association with HTTP by intercepting the TLS authentication sent from the application to the client. The attacker sends a decoded form of the application's site to the client while maintaining the anchored session with the application. In the meantime, the client's entire session is open to the attacker.
+
+### Types of Man-in-the-Middle Attacks
+- [Wi-Fi Eavesdropping](https://doubleoctopus.com/security-wiki/threats-and-tools/wi-fi-eavesdropping/) - Attackers set up Wi-Fi connections with very legitimate-sounding names, using similar names to a business nearby or evil twin tactics (when hackers mimic trusted public WiFi connections in the area). It will trick you into thinking that it is the same network that you have used in the past. Once a user connects to the attacker’s Wi-Fi, he/she will be able to watch the user’s online activity and intercept important information, such as login credentials and payment information.
+- [Email Hijacking](https://doubleoctopus.com/security-wiki/threats-and-tools/email-hijacking/) - Attackers sometimes target email accounts of banks and other financial institutions. Once they gain access, they can check transactions between the institution and its customers. After that, they can spoof the institutions’s email address and send their own instructions to customers. This fools the customers into following the attacker’s instructions rather than the legitimate institution.
+- [Session Hijacking](https://owasp.org/www-community/attacks/Session_hijacking_attack) - Many websites store a “session browser cookie” on the user’s machine. This cookie is invalidated when the user logs off. However, while the session is active, the cookie provides identity, access, and tracking information. A Session Hijack occurs when an attacker steals a session cookie. This can happen if the user’s machine is infected with malware or browser hijackers. It can also happen when an attacker uses a cross-site scripting (XSS) attack – where the attacker injects malicious code into a frequently-used website.
+
+### How to prevent Man-in-the-Middle attacks?
+- Make sure that the websites you visit have HTTPS in front of the URL.
+- Before clicking on emails, check the sender of the email.
+- If you’re a website admin, you should implement HSTS.
+- Set up an intrusion detection system (IDS).
+- DO NOT make a purchase or send sensitive data on a public Wi-Fi network. If doing so, be sure to use secure VPN.
+- If your website is using SSL, make sure you have disabled insecure SSL/TLS protocols. You should only have enabled TLS 1.1 and TLS 1.2
+- Don’t click on suspicious links or emails.
+- Avoid installing unnecessary software or plugins, especially those that offer something for free. This reduces the likelihood that you install something that can install a MitM attack.
```

Comment:
Should explain early in the article that `MitM` is the common abbreviation used.